### DISCUSSION TITLE
Timer Implementation

### DISCUSSION THREAD

**Discussion by DarthGecko** [vigoren/foundryvtt-simple-calendar#216] [Ideas]
Is there a way to write a macro for timers? Like a timer that goes off after a certain number of days has passed since an item gets used to notify that it has recharged?

**Comment by vigoren** [comment]
There is another module that does this kind of stuff, about-time.
https://gitlab.com/tposney/about-time

It requires Simple Calendar but provides an API to set up reminders or things that happen at specific times. I think it should be able to do what you are looking for!
